app/views/members/_roles.html.haml

```diff
@@ -0,0 +1,19 @@
+%h2
+  = t("navigation.show.roles_of") + " " + $member
+
+%table
+  %thead
+    %tr
+      %th= t('roles.role_title')
+      %th= t('roles.role_description')
+      %th
+        &nbsp;
+  - $roles.each do |role|
```

Comment:
Replace with @ (instance variable)

Reply:
This will not work. You need to modify the endpoints so it will take a memberid as input. This will be dependent on the URL, which I am not a fan of, for this specific use case.

Reply:
Doesn't the endpoint already accept the member ID as an input (line 13 in show_roles)? Changing the whole state of the server in this way seems very dangerous and should instead be done using params like every other endpoint.